docs/remote_inference_blueprints/bedrock_connector_titan_multimodal_embedding_blueprint.md

```diff
@@ -46,6 +49,21 @@ POST /_plugins/_ml/connectors/_create
         "content-type": "application/json",
         "x-amz-content-sha256": "required"
       },
+      "request_body": "{\"inputText\": \"${parameters.inputText:-null}\", \"inputImage\": \"${parameters.inputImage:-null}\"}",
+      "pre_process_function": "connector.pre_process.bedrock.multimodal_embedding",
+      "post_process_function": "connector.post_process.bedrock.embedding"
+    }
+  ]
+}
+```
+**Use painless script**
+```json
+POST /_plugins/_ml/connectors/_create
+{
+  ... //same above
```

Comment:
How about just show the complete request ?

Reply:
I think for most cases if users are using the default process function, this simplification can save time to scroll down to copy following parts.